scripts/issue/226/disable_borrow.py

```diff
@@ -0,0 +1,18 @@
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from brownie import interface
+from eth_abi import encode_abi
+
+
+def disable(ftoken_addr):
```

Comment:
suggesting to keep using `main` as function name if there is only one function in the file; prevents from having to specify the function manually.